wasmbinding/stargate_allowlist.go

```diff
@@ -49,6 +50,9 @@ func AcceptedStargateQueries() wasmkeeper.AcceptedStargateQueries {
 		"/neutron.interchaintxs.v1.Query/Params":                   &interchaintxstypes.QueryParamsResponse{},
 		"/neutron.interchaintxs.v1.Query/InterchainAccountAddress": &interchaintxstypes.QueryInterchainAccountAddressResponse{},
 
+		// cron
+		"/neutron.cron.Query/Params": &crontypes.QueryParamsResponse{},
```

Comment:
@pr0n00gler do we need cron stargate query? I mean I think we specifically avoided queries where we have binding queries?